InterviewPresenter.swift

```diff
@@ -0,0 +1,51 @@
+
+import UIKit
+import Foundation
+
+protocol ProfileViewInput: AnyObject {
+    var output: ProfileViewOutput? { get set }
+    
+    func setResult(value: String)
+}
+
+protocol ProfileViewOutput {
+    func viewIsReady()
+}
+
+enum TypeCalculate: String {
+    case sum
+    case difference
+}
+
+class Presenter: ProfileViewOutput {
+    weak var view: ProfileViewInput!
+    
+    var a = Int()
+    let b: Int
+    var c = 0
```

Comment:
Можно все параметры для расчета сделать константными
Тк они не изменяются
```suggestion
    let a, b, c: Int
```